### ISSUE TITLE
dovico's executable "console" conflicts with other gems

### ISSUE THREAD

**Issue by mehlah** [trainline-eu/dovico#12]
A `gem install dovico` gave me

```
Fetching: dovico-1.2.0.gem (100%)
dovico's executable "console" conflicts with countries
Overwrite the executable? [yN]  n
ERROR:  Error installing dovico:
	"console" from dovico conflicts with installed executable from countries
```

I guess renaming it `devico-console` (and renaming countries' one) would make the consoles world better.

**Comment by chtitux** [comment]
We should just not ship this file, only used for debug stuff.

**Comment by mehlah** [comment]
Closed by #14
